frontend/sw360-portlet/src/main/webapp/html/projects/includes/projects/clearingStatus.jspf

```diff
@@ -30,7 +31,7 @@
                 <th></th>
                 <th></th>
                 <th></th>
-                <th></th>
+                <th width="15%"></th>
```

Comment:
This looks like a hack, why is this necessary?

Reply:
Without width allocation the table is distorting and the buttons are coming vertically.On the same table ,there are two other table header for which width allocation is present and also observed the same for few other tables. Please suggest if there is any better approach for this.

Reply:
one solution is maybe outlined here (using css classes) https://stackoverflow.com/questions/16100956/prevent-after-element-from-wrapping-to-next-line